lib/view_model/utils/collections.rb

```diff
@@ -1,21 +1,27 @@
+# frozen_string_literal: true
+
 class ViewModel::Utils
   module Collections
     def self.count_by(enumerable)
-      enumerable.each_with_object(Hash.new(0)) do |el, counts|
-        key         = yield(el)
-        counts[key] += 1 unless key.nil?
+      enumerable.each_with_object({}) do |el, counts|
+        key = yield(el)
+
+        unless key.nil?
+          counts[key] ||= 0
```

Comment:
What's the motivation to rewrite this? Avoiding absent keys from having "0" value?

While you're here, the following is clearer, imo.
```ruby
counts[key] = (counts[key] || 0) + 1
```